src/wampy.js

```diff
@@ -1510,19 +1510,45 @@ class Wampy {
      *************************************************************************/
 
     /**
+     * DEPRECATED
```

Comment:
Please use JSDoc comments instead of clean text. This way most  IDEs will highlight this for developers.
Refer to: https://jsdoc.app/tags-deprecated.html